src/status_im/contexts/communities/events.cljs

```diff
@@ -228,10 +227,7 @@
   (when community-js
     {:db (update db :communities/fetching-communities dissoc community-id)
      :fx [[:dispatch [:communities/handle-community community-js]]
-          [:dispatch [:chat.ui/spectate-community community-id]]
-          [:dispatch
-           [:chat.ui/cache-link-preview-data (link-preview.events/community-link community-id)
-            (data-store.communities/<-rpc community-js)]]]}))
```

Comment:
This line here can be quite expensive in large communities and, unfortunately, we were doing the full conversion using `<-rpc` twice, once here, and again inside the `:communities/handle-community` handler. Ouch!